hera_cal/lstbin.py

```diff
@@ -672,10 +763,19 @@ def lst_bin_files(data_files, input_cals=None, dlst=None, verbose=True, ntimes_p
                             apply_cal.calibrate_in_place(data, gains, data_flags=flags, cal_flags=cal_flags,
                                                          gain_convention=uvc.gain_convention)
 
+                    # redundantly average baselines, keying to baseline group key
+                    # on earliest night.
+                    if average_redundant_baselines:
+                        if ignore_flags:
+                            raise NotImplementedError("average_redundant_baselines with ignore_flags True is not implemented.")
+                        utils.red_average(data=data, flags=flags, nsamples=nsamps,
```

Comment:
Does this function handle `include_autos` correctly?

Reply:
it does if you want the autos to be averaged together.